script.js

```diff
@@ -0,0 +1,43 @@
+
+
+API_KEY = "yTk65md4VUpLFHxJvS6bqFOcT31mDqFg"
+//api key from GHIPY 
+const formEl = document.querySelector('form');
+document.addEventListener("DOMContentLoaded", giphysearch);

```

Comment:
So does the DOMContentLoaded, load the content instantly onto the page? this is really interesting